pkg/command/command.go

```diff
@@ -79,7 +79,7 @@ func (c *Command) RunExecutable(executable string, params ...string) error {
 	}
 
 	if len(c.env) > 0 {
-		cmd.Env = c.env
+		setEnvironment(cmd, c.env)
```

Comment:
Do you still need the len(c.env) in line 81 as it is checked again in setEnvironment?

Reply:
yes, not needed anymore --> removed